src/types.ts

```diff
@@ -125,4 +125,10 @@ export interface Storage<T extends StorageValue = StorageValue> {
     base?: string,
     options?: { parents?: boolean }
   ) => { base: string; driver: Driver }[];
+  // Aliases
+  keys: Storage["getKeys"];
+  get: Storage["getItem"];
+  set: Storage["setItem"];
+  has: Storage["hasItem"];
+  del: Storage["removeItem"];
```

Comment:
How do you think about `remove` as alternative. This way we prevent abbreviation same as other new shortcuts

Reply:
Remove sounds good

Reply:
Added the `remove`, I like the `del` also as similar to Redis